framework/src/application/node/node.ts

```diff
@@ -474,6 +480,16 @@ export class Node {
 					params.data,
 					params.peerId,
 				),
+			getSchema: () => ({
+				account: this._chain.accountSchema,
+				baseBlockHeader: blockSchema,
```

Comment:
```suggestion
				blockHeader: blockSchema,
```

Reply:
The request in the issue was to use baseBlockHeader @shuse2 ?